updating triannual self reflection section (#36)
greenelab/onboarding: onboarding.md

- meeting abstracts: submitted/accepted
- software releases
- other honors
- prior goal review: Did you achieve the goals you set in the last cycle?
- goals for next session: What would you like to accomplish by the end of next cycle?
- self-reflection: What do you regard as your strengths and as areas where you need improvement?

The report should be in the form of a plain text file, markdown file, or PDF and the file should be called lastname-reflection-yearmonth (e.g. Greene-reflection-202506.txt).
Submit the report in an email to Casey with subject line "\[Full name\] Triannual Reflection \[Month\], \[Year\]."
These reflections are due on February 1, June 1, and October 1 of each year. 

> University of Colorado postdocs have an annual contract renewal process in late January, which will count for their February 1 triannual self reflection.

Reviewer: ```suggestion
University of Colorado postdocs have an annual contract renewal process in late January, which can be used in place of the February 1 triannual self-reflection.

```

- self-reflection. What do you regard as your strengths and as areas where you need improvement?

The report should be in the form of a plain text file, markdown file, or PDF and the file should be called lastname-reflection-yearmonth (e.g. Greene-reflection-201908.txt).
Submit the report in a direct message to Casey via slack.
During the summer, graduate students are required to complete an individual development plan (IDP).

> Post-docs must complete an IDP prior to their annual contract renewal.

Reviewer 2: This may still be helpful for the postdoc contract renewal process. If there is another mechanism specific to CU, we should add it to this document. I would do this, but I haven't yet gone through the process.

Reviewer 3: I went through the review process at the end of January. It's basically in the middle Dec 1 and Apr 1 reflections... But I think the self reflections will still help with that process.
